ivy_tests/test_ivy/test_frontends/test_paddle/test_linalg.py

```diff
@@ -688,6 +699,49 @@ def test_paddle_eigvalsh(
     )
 
 
+# diagonal
+@handle_frontend_test(
+    fn_tree="paddle.linalg.diagonal",
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
I guess this should be changed to `valid`, you may do the necessary changes as well